internal/completions/httpapi/handler.go

```diff
@@ -255,7 +255,8 @@ func newStreamingResponseHandler(logger log.Logger, db database.DB, feature type
 			l.Error("attribution error", log.Error(err))
 		}
 		f := guardrails.NoopCompletionsFilter(eventSink)
-		if featureflag.FromContext(ctx).GetBoolOr("autocomplete-attribution", false) {
+		if conf.GetConfigFeatures(conf.SiteConfig()).Attribution &&
```

Comment:
it looks like GetConfigFeatures is a bit of a naughty API and can return nil. Let me make this more robust for ya so we can get this in before our release branch shutsdown a bit more.
```suggestion
		if cf := conf.GetConfigFeatures(conf.SiteConfig()); cf != nil && cf.Attribution &&
```